pkg/pkg.ml

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ocaml
+#use "topfind"
+#require "topkg"
+open Topkg
+
+
+let () =
+  Pkg.describe "tuntap" @@ fun c ->
+  Ok [
+    Pkg.mllib "lib/tuntap.mllib";
+    Pkg.clib "lib/libtuntap_stubs.clib";
+    Pkg.test "test/getifaddrs_test";
```

Comment:
Thanks a lot for the commit! Had to read the ocamlbuild manual to understand what was going on. Where did `lib/libtuntap_stubs.a` come from? Is that one of the output files from `Pkg.clib "lib/libtuntap_stubs.clib"`? The `.a` extension is the C equivalent of `.cma` in OCaml? I have minimal knowledge of C and have just been learning it recently.

Btw, can you push your commit into this PR or would you like me to copy the code from there @hannesm?